src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -281,6 +282,18 @@ given baseline hash doesn't exist.
   in
   Arg.value (Arg.opt Arg.(some string) None info)
 
+let o_diff_depth : int Term.t =
+  let info =
+    Arg.info [ "diff-depth" ]
+      ~doc:
+        {|The depth of the differential scan, the number of steps (both in the
+       caller and callee sides) from the targets in the call graph tracked
+       by the deep preprocessor. Only applied in differential scan mode.
+       Default to 2.
+       |}
```

Comment:
Same feedback here as for the message in pysemgrep

Reply:
done